app/models/patient.rb

```diff
@@ -38,7 +38,8 @@ class Patient
   embeds_many :notes
   belongs_to :pledge_generated_by, class_name: 'User', inverse_of: nil
   belongs_to :pledge_sent_by, class_name: 'User', inverse_of: nil
-
+  belongs_to :last_edited_by, class_name: 'User', inverse_of: nil
```

Comment:
I was initially on the fence about this but the more I think about it the more I like it. Good call on this! It's a good workaround to the problem of not having `current_user` in a model, it updates the stuff in its own db transaction, and we get a fringe benefit of having this in one convenient place if we need it. (We store something similar now but it's buried in the audit history, and wouldn't be available at the time the before_update callback fired.)